pipelines/fast-test.yml

```diff
@@ -179,14 +179,14 @@ stages:
         set -e
         cd test
         python -m pytest ut --cov-config=.coveragerc \
-          --ignore=ut/sdk/test_pruners.py \
-          --ignore=ut/sdk/test_compressor_tf.py \
-          --ignore=ut/sdk/test_compressor_torch.py \
-          --ignore=ut/sdk/test_model_speedup.py
-        python -m pytest ut/sdk/test_pruners.py --cov-config=.coveragerc --cov-append
-        python -m pytest ut/sdk/test_compressor_tf.py --cov-config=.coveragerc --cov-append
-        python -m pytest ut/sdk/test_compressor_torch.py --cov-config=.coveragerc --cov-append
-        python -m pytest ut/sdk/test_model_speedup.py --cov-config=.coveragerc --cov-append
+          --ignore=ut/compression/v1/test_pruners.py \
+          --ignore=ut/compression/v1/test_compressor_tf.py \
+          --ignore=ut/compression/v1/test_compressor_torch.py \
+          --ignore=ut/compression/v1/test_model_speedup.py
+        python -m pytest ut/compression/v1/test_pruners.py --cov-config=.coveragerc --cov-append
+        python -m pytest ut/compression/v1/test_compressor_tf.py --cov-config=.coveragerc --cov-append
+        python -m pytest ut/compression/v1/test_compressor_torch.py --cov-config=.coveragerc --cov-append
+        python -m pytest ut/compression/v1/test_model_speedup.py --cov-config=.coveragerc --cov-append
```

Comment:
I am wondering why v2 is not added here?

Reply:
In v2 we don't use big models and separate execution is to avoid the issue of GPU memory release, so v2 is not added here.